Resources/ProtobufGen/gc.proto

```diff
@@ -1,21 +1,21 @@
-enum GCProtoBufMsgSrc {

```

Comment:
This file changed for no reason (line endings I assume?)

Reply:
I only moved the files... possibly something in my git install tweaked the line endings?